### PR TITLE
Sync Status: Update UI

### PR DESCRIPTION
**Pull request by scolsen**
Modify the sync status/history UI to better reflect the latests designs. Also refactors our approach to passing data over to the list adapter slightly, and adds some new drawable resources.

Screenshot 2024-02-15 at 3 54 37 PM

@gino-m FYI - but I think we still need a few tweaks on the UI itself?

### CODE REVIEW

**Comment by scolsen** [comment]
See #876

File: ground/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -69,7 +69,7 @@
   <string name="date">Date</string>
   <string name="time">Time</string>
   <!-- Other strings (to be organized) -->
-  <string name="sync_status">History and sync status</string>
+  <string name="sync_status">History</string>
```
Comment (gino-m) [inline ground/src/main/res/values/strings.xml:72]: The title in the nav menu should still be "History and sync status". @vittorino  Was the different labels between that and the status screen header intentional?

**Review by gino-m — APPROVED** [review]
Nice! I assume the correct colors and layout will come later, so LGTM for this change, modulo @vittorino 's guidance on the screen title and failing checks.

**Comment by vittorino** [comment]
I think we should keep the entire "History and sync status" if that's what's in the nav bar. In very small screens the header may break, so I'd also recommend we shorten this title if we can. Maybe just "history and sync"?

**Comment by gino-m** [comment]
> I think we should keep the entire "History and sync status" if that's what's in the nav bar. In very small screens the header may break, so I'd also recommend we shorten this title if we can. Maybe just "history and sync"?

Good point. @amegantz Can we just say "Sync status" in the screen title and side menu? History is implied.

**Pull request merged**